addons/web/static/src/legacy/scss/kanban_column_progressbar.scss

```diff
@@ -110,6 +110,7 @@
     @include o-kanban-css-filter(success, theme-color('success'));
     @include o-kanban-css-filter(warning, theme-color('warning'));
     @include o-kanban-css-filter(danger, theme-color('danger'));
+    @include o-kanban-css-filter(info, theme-color('info'));
```

Comment:
We have to see with the @odoo/rd-framework-js if they agree with this. 🙂 
I guess yes. 😇